helix.el

```diff
@@ -211,6 +213,12 @@ point.  Otherwise, continue the existing region."
   (helix--clear-highlights)
   (end-of-line))
 
+(defun helix-go-first-non-blank-in-line ()
+  "Go to first non blank character in line."
```

Comment:
Minor nitpick, but I'd prefer "non-whitespace" to blank, since that matches the Emacs & Helix documentation strings.

- Function name: `helix-go-first-nonwhitespace`
- Docstring: "Go to first non-whitespace character in line."